### ISSUE TITLE
Invalid Preset when Calculating the quality achieved with different presets

### ISSUE THREAD

**Issue by InB4DevOps** [BassThatHertz/video-quality-metrics#3]
Tried this:
`python video-metrics.py -ovp test.mp4 -e x264 -p medium fast faster -crf 18 -ssim -psnr`

Got this:

```
-----------------------------------------------------------------------------------------------------------
If the path contains a space, the path argument must be surrounded in double quotes. Example:
python video-metrics.py -ovp "C:/Users/H/Desktop/test file.mp4" -p fast veryfast
For more information, enter 'python video-metrics.py -h'
-----------------------------------------------------------------------------------------------------------
File: test.mp4
Framerate: 23.976023976023978 FPS
More than one CRF value specified. CRF comparison mode activated.
CRF values [18] will be compared and the ['medium', 'fast', 'faster'] will be used.
-----------------------------------------------------------------------------------------------------------
Transcoding the video with CRF 18...
Guessed Channel Layout for Input Stream #0.1 : 6.1
x264 [error]: invalid preset 'mediumfastfaster'
[libx264 @ 00000268bf4f00c0] Error setting preset/tune mediumfastfaster/(null).
Error initializing output stream 0:0 -- Error while opening encoder for output stream #0:0 - maybe incorrect parameters such as bit_rate, rate, width or height
```

**Comment by InB4DevOps** [comment]
The problem here is that the script falsely evaluates that CRF comparison mode is activated.

**Comment by CrypticSignal** [comment]
Fixed in commit df58b05.
